SPECS/php/php.spec

```diff
@@ -10,22 +9,15 @@
 # Please preserve changelog entries
 #
 # API/ABI check
-
 %global apiver      20190902
 %global zendver     20190902
 %global pdover      20170320
-
 # Adds -z now to the linker flags
 %global _hardened_build 1
```

Comment:
Another ignorable small request: we have `_hardened_build` enabled for all specs by default, so we can remove such lines. I doubt we'd ever want to disable this macro globally, so there's no harm keeping it the way it is, it's just minor clean-up.